### PR TITLE
Skeleton for the contribution flow update

### PR DESCRIPTION
**Pull request by sbinlondon**
Relates to the [new contribution flow project](https://github.com/opencollective/opencollective/projects/39)

Handing over for @Betree to continue work next week.

Set out as much of the outline of the project as I could.

Stuff like the header, success page, navigation buttons, and steps progress are more done than the rest.

There's still tons to do like writing GraphQL queries, making new components, most of the functionality, etc, but hopefully this is a good structure to start working within!

### CODE REVIEW

**Comment by Betree** [comment]
Require https://github.com/opencollective/opencollective-api/pull/4243

File: pages/new-contribution-flow.js [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+              totalDonations {
+                value
+                currency
+              }
+            }
+          }
+        }
+      }
+    }
+  }
```
Comment (Betree) [inline pages/new-contribution-flow.js:272]: I think we can remove that part now (can be done in a followup PR)

```suggestion
```

**Comment by Betree** [comment]
Ah looks like I removed too many brackets in my suggestion. Sorry!

**Comment by sbinlondon** [comment]
> Ah looks like I removed too many brackets in my suggestion. Sorry!

It's ok I can pull it and fix it

**Pull request merged**